Metagenomics/tutorials/mothur-miseq-sop.md

```diff
@@ -0,0 +1,1516 @@
+create a collection now:
+
+> ### :pencil2: Hands-on: Organizing our data into a collection
+>
+> Since we have paired-end data, each sample consist of two separate fastq files, one containing the
+> forward reads, and one containing the reverse reads. We can recognize the pairing from the file names,
+> which will differ only by `_R1` or `_R2` in the filename. We can tell Galaxy about this paired naming
+> convention, so that our tools will know which files belong together.
+>
+> - Click on the **checkmark icon** at top of your history.
+>   ![](../../shared/images/history_menu_buttons2.png)
+> - Select all the fastq files (40 in total), then click on **for all selected..** and select
+>   **Build List of Dataset Pairs** from the dropdown menu.
+> - In the next dialog window you can create the list of pairs. By default Galaxy will look for pairs
+>   of files that differ only by a `_1` and `_2` part in their names. In our case however, these
+>   should be `_R1` and `_R2`. Please change these values accordingly. You should now see a list of
+>   pairs suggested by Galaxy,
+>
+>   ![](../images/create_collection.png)
+>
+> - Examine the pairings, if it looks good, you can click on **auto-pair** to create the suggested
+>   pairs.
+>   ![](../images/create_collection2.png)
+>
+>   The middle segment is the name for each pair. You can change these names by clicking on them. These
+>   names will be used as sample names in the downstream analysis so always make sure they are
+>   informative.
+>
+> - Once you are happy with your pairings, enter a name for your new collection at the bottom right of
+>   the screen. Then click the **Create List** button. A new dataset collection item will now appear in
+>   your history.
+{: .hands_on}
+
+
+# Quality Control
+
+## Reducing sequencing and PCR errors
+
+The first thing we want to do is combine our forward and reverse reads for each sample. This is done
+using the make.contigs command, which requires the paired collection as input. This command will extract
+the sequence and quality score data from your fastq files, create the reverse complement of the reverse
+read and then join the reads into contigs. Then we will combine all samples into a single fasta file,
+remembering which reads came from which samples using a *group* file.
+
+> ### :nut_and_bolt: Algorithm details
+> We have a very simple algorithm to do this. First, we align the pairs of sequences. Next, we look
+> across the alignment and identify any positions where the two reads disagree. If one sequence has a
+> base and the other has a gap, the quality score of the base must be over 25 to be considered real. If
+> both sequences have a base at that position, then we require one of the bases to have a quality score
+> 6 or more points better than the other. If it is less than 6 points better, then we set the consensus > base to an N.
+{: .comment}
+
+
+> ### :pencil2: Hands-on: Merging samples
```

Comment:
could do, just figured we'd stay within the mothur suite (this is a mothur tool, may not have been clear), its basically does the same thing